ibu-imager/cmd/create.go

```diff
@@ -61,9 +64,14 @@ func create() {
 
 	// Check if containerRegistry was provided by the user
 	if containerRegistry == "" {
-		fmt.Printf(" *** Please provide a valid container registry to store the created OCI images *** \n")
-		log.Info("Skipping OCI image creation.")
-		return
+		log.Printf(" *** Please provide a valid container registry to store the created OCI images *** ")
+		log.Fatal("Skipping OCI image creation.")
+	}
+
+	// Check if recertContainerImage was provided by the user
+	if recertContainerImage == "" {
```

Comment:
In case we have this new param we should change 
https://github.com/openshift-kni/lifecycle-agent/blob/main/ibu-imager/installation_configuration_files/scripts/installation-configuration.sh#L32
make it template and set image to it before copying this file to the seed image

Reply:
This is being addressed in #47.